core/controllers/editor.py

```diff
@@ -103,8 +103,6 @@ def get(self, exploration_id):
                 self.user_id and not has_seen_editor_tutorial)
             exploration_data['show_state_translation_tutorial_on_load'] = (
                 self.user_id and not has_seen_translation_tutorial)
-            exploration_data['exploration_is_linked_to_story'] = bool(
-                exp_services.get_story_id_linked_to_exploration(exploration_id))
```

Comment:
Oh no, don't remove this. I actually use this in my PR [here](https://github.com/oppia/oppia/pull/12233)

Reply:
Done!